apps/omg_status/lib/omg_status/application.ex

```diff
@@ -31,6 +31,14 @@ defmodule OMG.Status.Application do
         []
       end
 
+    # if there's no SENTRY_DSN (local Watcher for example)
+    # don't attach Sentry as backend logger to report issues to
+    _ =
+      case System.get_env("SENTRY_DSN") do
+        nil -> :ok
+        _ -> {:ok, _} = Logger.add_backend(Sentry.LoggerBackend)
+      end
+
```

Comment:
🙏 ❤️  Thanks for the fix!

Also, is this the best place for us to load this? Why not [omg/application.ex](https://github.com/omisego/elixir-omg/blob/master/apps/omg/lib/omg/application.ex)?

Reply:
the way umbrella applications are started is that the VM figures out what the dependency hierarchy is. Omg status is more or less at the top because it doesn't have any dependencies but it's a dependency to many - so it'll start as one of the first. And if you're adding a logger backend - you would want to add it as soon as possible so that you're able to catch exits that happen while applications are starting.

Reply:
there's another one... the macro endpoint hooks work independently of logger backend
```
use Sentry.Phoenix.Endpoint
```
need to check how to enable/disable these